package-list.json

```diff
@@ -6,24 +6,26 @@
     "list": [
       {
         "version": "current",
-        "descmd": "Continuous Integration Build (latest in version control)",
-        "path": "https://build.fhir.org/ig/HL7/fhircast-docs",
+        "path": "'http://hl7.org/fhir/uv/fhircast-docs",
```

Comment:
All of these additions contradict the HL7 documentation, but also don't make sense ...

Reply:
but trying it anyway ... there's too many conflicts to merge this PR :(

Reply:
setting a ci-build's path to hl7.org really doesn't make sense, though.